### PR TITLE
chore(ci): auto deploy gnochess realm

### PR DESCRIPTION
**Pull request by albttx**

### CODE REVIEW

**Comment by thehowl** [comment]
does this change the .env environment variable? (GNO_CHESS_REALM iirc)

also shouldn't netlify re-deploy everything (including the gno.land node) when we commit master?

File: .github/workflows/deploy-realm.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+on:
+  push:
+    paths:
+      - "realm/**"
+    branches:
+      - main
+
+env:
+  GNOKEY: "go run github.com/gnolang/gno/gno.land/cmd/gnokey"
+  GNOCHESS_REALM: gno.land/r/demo/chess_${GITHUB_SHA}
```
Comment (moul) [inline .github/workflows/deploy-realm.yml:12]: would be nice to deploy based on github_pr once merged too.

Comment (moul) [reply .github/workflows/deploy-realm.yml:12]: hum, googled and can't find an easy way of getting the PR number of a merged commit, only when merging on master/main. :/

**Review by moul — APPROVED** [review]

**Comment by albttx** [comment]
@thehowl sorry for late reply, we discussed about it during yesterday's call.
It was the plan, but we said it could be messed up for testers so we discussed with @alexiscolin about a system for selecting the realm we want to use more dynamically 

In the first time, we can merge this

**Pull request merged**